server/src/test/java/org/apache/druid/server/coordinator/duty/KillUnusedSegmentsTest.java

```diff
@@ -723,6 +901,17 @@ private void validateLastKillStateAndReset(final String dataSource, @Nullable fi
     overlordClient.deleteLastKillInterval(dataSource);
   }
 
+  private void createAndAddUsedSegment(final String dataSource, final Interval interval, final String version)
+  {
+    final DataSegment segment = createSegment(dataSource, interval, version);
+    try {
+      SqlSegmentsMetadataManagerTestBase.publishSegment(connector, config, TestHelper.makeJsonMapper(), segment);
+    }
+    catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
   private void createAndAddUnusedSegment(
```

Comment:
Nit: This method can now call the `createAndAddUsedSegment` as that part is common.